docs/ADVANCED-OPTIONS.md

```diff
@@ -97,3 +97,36 @@ Refer to the demo [`NavDestination.kt`](../demo/src/main/kotlin/dev/hotwire/turb
 You may encounter situations where a truly single-`Activity` app may not be feasible. For example, you may need an `Activity` for logged-out state and a separate `Activity` for logged-in state.
 
 In such cases, you need to create an additional `Activity` that also implements the `TurboActivity` interface. You will need to be sure to register each `Activity` by calling [`TurboSessionNavHostFragment.registeredActivities()`](../turbo/src/main/kotlin/dev/hotwire/turbo/session/TurboSessionNavHostFragment.kt) so that you can navigate between them.
+
+## Enable Debug Logging
+You may encounter some time when you need to see what turbo-android is doing behind the scenes in these instances you can enable debug logging by overriding the `onSessionCreate` method in your `NavHostFragment` and turning it on. For example:
```

Comment:
Suggested tweaks:

During development,  it may be helpful to see what `turbo-android` is doing behind the scenes. To enable debug logging, override the `onSessionCreated()` method in your `NavHostFragment` and call `session.setDebugLoggingEnabled(true)`. Debug logging should always be disabled in your production app. For example: